odata_query/sqlalchemy/sqlalchemy_core.py

```diff
@@ -0,0 +1,380 @@
+import operator
```

Comment:
This looks extremely similar to the ORM visitor (which is a good thing!).
I might refactor these to have a common base class.

Reply:
It is indeed, yes sharing a base makes absolute sense!